sdks/how-to/how-to-manage-healthcare-elements.md

```diff
@@ -4,3 +4,283 @@ description: Learn how to manage healthcare elements
+
+<!-- file://code-samples/how-to/manage-healthcare-elements/index.mts snippet:create multiple related HEs as data owner-->
+```typescript
+const startHealthcareElement = await api.healthcareElementApi.createOrModifyHealthcareElement(
+  new HealthcareElement({
+    description: 'The patient has been diagnosed Pararibulitis',
+    codes: new Set([
+      new CodingReference({
+        id: 'SNOMEDCT|617|20020131',
+        type: 'SNOMEDCT',
+        code: '617',
+        version: '20020131',
+      }),
+    ]),
+    openingDate: new Date('2019-10-12').getTime(),
+  }),
+  patient.id,
+)
+
+const followUpHealthcareElement = await api.healthcareElementApi.createOrModifyHealthcareElement(
+  new HealthcareElement({
+    description: 'The patient recovered',
+    openingDate: new Date('2020-11-08').getTime(),
+    healthcareElementId: startHealthcareElement.healthcareElementId,
+  }),
+  patient.id,
+)
+```
+
+:::note
+
+The `healthcareElementId` is the id of the first Healthcare Element of the series.
+
+:::
+
+Several unrelated Healthcare Elements can also be created at once.
+
+<!-- file://code-samples/how-to/manage-healthcare-elements/index.mts snippet:create multiple HEs as data owner-->
+```typescript
+const healthcareElement1 = new HealthcareElement({
+  description: 'The patient has been diagnosed Pararibulitis',
+  codes: new Set([
+    new CodingReference({
+      id: 'SNOMEDCT|617|20020131',
+      type: 'SNOMEDCT',
+      code: '617',
+      version: '20020131',
+    }),
+  ]),
+  openingDate: new Date('2019-10-12').getTime(),
+})
+
+const healthcareElement2 = new HealthcareElement({
+  description: 'The patient has also the flu',
+  openingDate: new Date('2020-11-08').getTime(),
+})
+
+const newElements = await api.healthcareElementApi.createOrModifyHealthcareElements(
+  [healthcareElement1, healthcareElement2],
+  patient.id,
+)
+```
+
+:::caution
+
+Even if you associate a Healthcare Element to a Patient, the Patient does not have access to it unless you give access 
+to them.
+
+:::
+
+
+
+## Sharing a Healthcare Element with a Patient
+
+After creating the Healthcare Element, the Healthcare Professional shares it with the Patient.
+
+<!-- file://code-samples/how-to/manage-healthcare-elements/index.mts snippet:HE sharing with data owner-->
+```typescript
+const sharedHealthcareElement = await api.healthcareElementApi.giveAccessTo(
+  healthcareElement,
+  patient.id,
+)
+```
+
+If the operation is successful, the method returns a Promise with the updated Healthcare Element.
+
+## Retrieving a Healthcare Element Using its ID
+
+A single Healthcare Element can be retrieved from the Backend using its id.
+
+<!-- file://code-samples/how-to/manage-healthcare-elements/index.mts snippet:retrieve a HE as data owner-->
+```typescript
+const healthcareElementToRetrieve = new HealthcareElement({
+  description: 'To retrieve, I must create',
+})
+
+const createdHealthcareElement = await api.healthcareElementApi.createOrModifyHealthcareElement(
```

Comment:
I would not put the create in the code sample : Reuse the one you previously created in the other code samples